src/options.rs

```diff
@@ -61,6 +61,8 @@ pub struct Options {
     ///   * >= 2 – suppress startup except for auth data, if present
     ///   * >= 3 – suppress all startup messages
     pub loglevel: LogLevel,
+    /// Whether to handle WebDAV requests. Default: true
```

Comment:
This doesn't match up with the description, should be false by default

Reply:
Fixed in c9d200c31c8392d74402ed3201e312f2796a1d9b